src/client/datascience/serviceRegistry.ts

```diff
@@ -200,33 +197,28 @@ import { WebviewExtensibility } from './webviewExtensibility';
 // README: Did you make sure "dataScienceIocContainer.ts" has also been updated appropriately?
 
 // eslint-disable-next-line
-export function registerTypes(serviceManager: IServiceManager, inNotebookApiExperiment: boolean, inCustomEditorApiExperiment: boolean) {
-    const usingCustomEditor = inCustomEditorApiExperiment && !vscode.env.appName.includes('Insider'); // Don't use app manager in case it's not available yet.
-    const useVSCodeNotebookAPI = inNotebookApiExperiment && !usingCustomEditor;

```

Comment:
I think these still need to be mutually exclusive. Otherwise we'll register both.